packages/xo-server/src/api/pool.mjs

```diff
@@ -116,6 +116,45 @@ setPoolMaster.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function disableHa({ pool }) {
+  await this.getXapi(pool).disableHa()
+}
+
+disableHa.params = {
+  pool: {
+    type: 'string',
+  },
+}
+
+disableHa.resolve = {
+  pool: ['pool', 'pool', 'administrate'],
+}
+
+// -------------------------------------------------------------------
+
+export async function enableHa({ pool, heartbeatSrs, configuration }) {
+  await this.getXapi(pool).enableHa(heartbeatSrs, configuration)
```

Comment:
The XAPI call still expects refs, though, so you need to resolve the SR IDs and pass their XAPI refs.